editor/pot_generator.cpp

```diff
@@ -141,7 +141,8 @@ void POTGenerator::_write_to_pot(const String &p_file) {
 
 			// Write context.
 			if (!context.is_empty()) {
-				file->store_line("msgctxt " + context.c_escape().quote());
+				// Escaping `'` is unnecessary, and is incompatible with translation tools like Weblate.
+				file->store_line("msgctxt " + context.c_escape().replace("\\'", "\'").quote());
```

Comment:
We have `json_escape` which is like `c_escape` but doesn't escape `'` and `\a`, whatever that is. That might work here?

Reply:
> We have `json_escape` which is like `c_escape` but doesn't escape `'` and `\a`, whatever that is. That might work here?

according to [wikipedia](https://en.wikipedia.org/wiki/Bell_character) `\a` is a bell character. i am not sure what it is used for nowadays, but i don't think it will be a problem.

Reply:
Done.